conjur/login/credential_data.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialData module
+
+This module represents an object that holds netrc data
+"""
+
+# pylint: disable=too-few-public-methods
+class CredentialData:
+    """
+    Used for setting user input data to login to Conjur
+    """
+
+    def __init__(self, machine=None, login=None, password=None):
+        self.machine = machine
+        self.login = login
+
+        # Purposefully not named the same for clarity in code. .netrc file format standard
+        # uses password but this value is actually the user/host api key. This convention should
+        # be kept. See https://www.labkey.org/Documentation/wiki-page.view?name=netrc especially
+        # the 'Use API Keys' section
+        self.password = password
```

Comment:
It could be None if the user didn't provide a password but the user wouldnt get past  `get_api_key()` which will fail on login and the netc won't be created